ow_plexil/src/plexil-adapter/PlexilPlanner.h

```diff
@@ -0,0 +1,32 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#ifndef Plexil_Planner_H
+#define Plexil_Planner_H
+
+#include <ros/ros.h>
+#include <ow_plexil/PlannerCommand.h>
+#include "OwExecutive.h"
+#include "OwInterface.h"
+
+class PlexilPlanner{
+  public:
+    PlexilPlanner();
```

Comment:
Since the default constructor does nothing, you could just declare this:
```
PlexilPlanner() = default;
```
Also, it's good to be explicit in every class about the other compiler-generated functions.  For this class, the appropriate declarations would be:
```
~PlexilPlanner() = default;
PlexilPlanner& PlexilPlanner (const PlexilPlanner&) = delete;
PlexilPlanner& operator=(const PlexilPlanner&) = delete;
```
assuming you don't want copying and assignment.
Oh, and rename "Planner" :)